### DISCUSSION TITLE
Command

### DISCUSSION THREAD

**Discussion by AlexMinecraft2012** [HbmMods/Hbm-s-Nuclear-Tech-GIT#717] [General]
Don't use /dumpthreadsandcrashgame

**Comment by TheForge129** [comment]
Why not, does it do exactly what it says it does?
💀

**Reply by AlexMinecraft2012** [reply]
Because /dumpthreadsandcrashgame crash it force closes minecraft

**Reply by TheForge129** [reply]
> Because /dumpthreadsandcrashgame crash it force closes minecraft

Read it.
"dumpthreadsand**crashgame**"

lmao

**Reply by OsakaLinuxScraper** [reply]
Wow you are a genius.

_this action was performed by a bot, to opt out, please comment "nobot"_

**Comment by SoB1nary** [comment]
are you ten
